File: Modul 2/C/PRAK204-2310817210013-Muhammad_Aufa_Fitrianda.c
#include <stdio.h>
#include <math.h>
#define pi 3.142857

void main(){
    double jari2, tinggi;
    scanf("%lf %lf", &jari2, &tinggi); 
    double volume = pi * pow(jari2, 2) * tinggi;
    double luas = 2 * pi * jari2 * (jari2 + tinggi);
    double keliling = 2 * pi * jari2; 

    printf ("\nVolume = %.2lf\n", volume);
    printf ("Luas = %.2lf\n", luas);
    printf ("Keliling = %.2lf\n", keliling); 
}

File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_2/Bahasa_Python/PRAK204-2310817210013-Muhammad_Aufa_Fitrianda.py
angka = input().split()
phi = 22/7
volume = phi * float(angka[0]) * float(angka[0]) * float(angka[1])
luas = 2 * phi * float(angka[0]) * (float(angka[0]) + float(angka[1]))
keliling = 2 * phi * float(angka[0])
print(f"Volume \t = {volume:.2f}")
print(f"Luas \t = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")